apps/Starship Prompt/uninstall

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sh -c 'rm "$(command -v 'starship')"' || error 'Failed to remove starship'
+
+#HOW CAN I REMOVE TEXT FROM ~/.bashrc????
```

Comment:
in general, I would suggest not adding anything to .bashrc, but I see no way around it in this case

you can remove matching text like this
```
sed -i '/\eval "$(starship init bash)"/d' "$HOME/.bashrc"
```